pymc/gp/gp.py

```diff
@@ -218,7 +238,7 @@ def conditional(self, name, Xnew, given=None, jitter=JITTER_DEFAULT, **kwargs):
         Xnew: array-like
             Function input values.
         given: dict
-            Can optionally take as key value pairs: `X`, `y`, `noise`,
+            Can optionally take as key value pairs: `X`, `y`,
```

Comment:
Do we need to replace `noise` by `sigma` here?

Reply:
This was for the latent class that doesn't use noise. I believe this was a mistake to begin with. Let me know if I am wrong

Reply:
Great, thanks for pointing it out :)